pkgs/desktops/gnome-3/core/mutter/default.nix

```diff
@@ -55,6 +55,10 @@ stdenv.mkDerivation rec {
       src = ./fix-paths.patch;
       inherit zenity;
     })
+    (fetchpatch {
+      url = "https://gitlab.gnome.org/GNOME/mutter/commit/8307c0f7ab60760de53f764e6636893733543be8.diff";
```

Comment:
Please describe why we have this patch in a comment.

Reply:
Good idea, added!